mne_bids/read.py

```diff
@@ -383,14 +394,32 @@ def read_raw_bids(bids_fname, bids_root, extra_params=None,
         # Get coordinate frames that electrode coordinates are in
         if kind == "meg":
             coord_frame = coordsystem_json['MEGCoordinateSystem']
+            coord_unit = coordsystem_json['MEGCoordinateUnits']
+        elif kind == 'eeg':
+            coord_frame = coordsystem_json['EEGCoordinateSystem']
+            coord_unit = coordsystem_json['EEGCoordinateUnits']
+
+            if coord_frame.lower() not in ['captrak', 'besa']:
+                warn("Defaulting to 'head' coordinate frame for "
+                     "EEG montage because mne-bids does not "
+                     "support {} coordinate frame yet.".format(coord_frame))
+            coord_frame = 'head'
         elif kind == "ieeg":
-            coord_frame = coordsystem_json['iEEGCoordinateSystem']
+            coord_frame = coordsystem_json['iEEGCoordinateSystem'].lower()
+            coord_unit = coordsystem_json['iEEGCoordinateUnits']
+
+            # default coordinate frames to available ones in mne-python
+            if coord_frame not in _ieeg_coordinate_frames_dict.keys():
```

Comment:
```suggestion
            if coord_frame not in _ieeg_coordinate_frames_dict:
```

Reply:
isn't it dangerous to convert to MRI when you don't know? To me this is bug prone and should rather raise an error

Reply:
So while that is true, iEEG only makes sense really if it is in the MRI coordinate frame. All derivatives of this, such as acpc, ras, etc. are still from the MRI image of the subject brain. I think a warning could suffice here?

If not, I would say it might be better to defer to 'unknown'? Since electrode coordinates can be present, and maybe the coordinatesystem is something like `LIA`, which isn't supported in mne-python yet, so the user would still at least have their montage set.

Reply:
For EEG, I believe we've prelim addressed it w/ the discussion down below.

For iEEG: https://github.com/mne-tools/mne-bids/pull/390#issuecomment-618462528